### PR TITLE
SEAR-186 Ability to filter search results by status

### PR DESCRIPTION
**Pull request by sunilb0575**
#### Which User story does this relate to (link)?
-https://osi-cwds.atlassian.net/browse/SEAR-186

#### Brief feature description
-Add multi-select dropdown with list of statuses.
1. Selecting more than one item shows number of items selected.
2. Selected items will be indicated by change of background color.

-Add checkbox to display facilities with active statuses.
1. When selected disables license status multi-select dropdown.

#### Does this PR block/effect other stories?
-No

#### Do any commands needs to be run in order to run the project with this PR?
-No

#### Code coverage percent diff as a result of this PR?
-No

### CODE REVIEW

**Comment by sunilb0575** [comment]
jenkins, retest this please

File: app/javascript/search/searchInput.js [hunk 1]
```diff
@@ -9,88 +11,115 @@ const SearchInput = ({
...
+            label='Facility Type'
+            id='facility_select'
+            selectClassName='searchSelect'
+            value={facilityTypeValue}
+            optionList={facilityTypes}
+            onChange={(event) => handleInputChange('facilityTypeValue', dictionaryNilSelectValue(event.target.options))} />
         </div>
-      </form>
-    </div>
+        <div className='col-xs-12 col-sm-3 col-md-3 col-lg-3'>
```
Comment (Nasir2712) [inline app/javascript/search/searchInput.js]: as discussed with this needs to be updated.

File: app/javascript/components/common/multiSelect.js [hunk 2]
```diff
@@ -3,35 +3,51 @@ import Select from 'react-select'
...
 MultiSelect.propTypes = {
   label: PropTypes.string,
   values: PropTypes.array,
+  disabled: PropTypes.bool,
+  searchable: PropTypes.bool,
+  clearable: PropTypes.bool,
   optionList: PropTypes.array,
+  valueRenderer: PropTypes.func,
+  options: PropTypes.func,
+  removeSelected: PropTypes.bool,
```
Comment (Nasir2712) [inline app/javascript/components/common/multiSelect.js]: add ```optionList: []``` in default props.

**Comment by Nasir2712** [comment]
retest this please

File: app/javascript/search/searchInput.js [hunk 3]
```diff
@@ -9,88 +11,115 @@ const SearchInput = ({
...
+            type='checkbox'
+            id='all_active'
+            gridClassName='allActive'
+            label='Active Status Only'
+            value={isAllActive}
+            checked={isAllActive}
+            onChange={(event) => (isAllActive ? (handleInputChange('isAllActive', false)) : (handleInputChange('isAllActive', true)))}
+          />
+        </div>
+        <div className='search_block col-xs-2 col-sm-2 col-md-2 col-lg-2'>
```
Comment (Nasir2712) [inline app/javascript/search/searchInput.js]: search_block css can be used on button itself.

File: app/javascript/search/searchInput.js [hunk 4]
```diff
@@ -9,88 +11,115 @@ const SearchInput = ({
...
+        </div>
+        <div className='col-xs-12 col-sm-2 col-md-2 col-lg-2' >
+          <BinarySelectorField
+            type='checkbox'
+            id='all_active'
+            gridClassName='allActive'
+            label='Active Status Only'
+            value={isAllActive}
+            checked={isAllActive}
+            onChange={(event) => (isAllActive ? (handleInputChange('isAllActive', false)) : (handleInputChange('isAllActive', true)))}
```
Comment (Nasir2712) [inline app/javascript/search/searchInput.js]: please use event to get rid of conditional

File: app/javascript/components/common/multiSelect.js (hunk 2 above)
Comment (sunilb0575) [reply app/javascript/components/common/multiSelect.js]: done.

**Comment by sunilb0575** [comment]
jenkins, retest this please

**Pull request merged**